### DISCUSSION TITLE
Estimated Amount

### DISCUSSION THREAD

**Discussion by JaSimps5** [CMSgov/hospital-price-transparency#97] [Q&A] [answered]
Regarding the estimated amount; I have read the CMS Cell Suppression policy that indicates estimates should only be provided if the data pool for that estimation are greater than 10 and a value of 999999999 when less than 10.  

Is there a time-frame in which we should be pulling those estimates from?  Past two years? Past year? Only since the beginning of the effective contract policy?

**Comment by RidenourK** [comment, marked as answer]
@JaSimps5. Thank you for your question. 

"Estimated allowed amount” is the average dollar amount that the hospital has historically received from a third-party payer for an item or service. Based on established industry practices CMS recommends, but does not require, incorporating one year of historic reimbursement data in the development of the estimated allowed amount.
